.claude/skills/playbook/SKILL.md

```
@@ -0,0 +1,368 @@
+---
+name: playbook
+description: Use this skill when writing, creating, or modifying bash playbook scripts in the playbooks/ directory. Activates for tasks involving server provisioning, site deployment, package installation, or any idempotent bash automation scripts.
+---
+
+# Playbook Development
+
+Playbooks are idempotent, non-interactive bash scripts that execute server tasks remotely. They receive context via environment variables and return YAML output.
+
+All rules MANDATORY. Bash style based on https://style.ysap.sh/md
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)